app.js

```diff
@@ -25,38 +25,39 @@ headSelect.addEventListener('change', () => {
 });
 
 middleSelect.addEventListener('change', () => {
-    // > set the character middle from the value of the middle <select>
-    // > redisplay the character
+    character.middle = middleSelect.value;
+    displayCharacter();
 });
 
 pantsSelect.addEventListener('change', () => {
-    // > set the character head from the value of the head <select>
-    // > redisplay the character
+    character.pants = pantsSelect.value;
+    displayCharacter();
 });
 
 function displayCharacter() {
     headImage.src = 'assets/character/' + character.head + '-head.png';
-    // > set the middle and pants images .src property based on the character middle and pants properties
+    middleImage.src = 'assets/character/' + character.middle + '-middle.png';
+    pantsImage.src = 'assets/character/' + character.pants + '-pants.png';
 }
 
 function displayPhrases() {
     // clear the list
     phraseList.innerHTML = '';
 
     for (const phrase of character.phrases) {
-        // > create an <li> element
-        // > set the text of the li to the phrase
-        // > append the <li> to the phrases list
+        const li = document.createElement('li');
+        li.textContent = phrase;
+        phraseList.append(li);
     }
 }
 
 addButton.addEventListener('click', () => {
-    // > Get the phrase from the value of the phrase input
-    // > Use ".push" to add the phrase to the end of the character phrases array
-    // > Re-display the phrases
-
-    // reset the input
-    phraseInput.value = '';
+    if (phraseInput.value !== '') {
```

Comment:
good job using a negative check to determine whether to take info from input and add to state.

here's a neat thing: so your ```if``` is checking whether or not something is true then performing what's in the code if it is. An empty string ```''``` is what's known as a falsy value, which means it's not equal to ```false``` but it acts the same as false. So if ```phraseInput.value === ''``` then ```phraseInput.value``` is falsy and ```if (phraseInput.value)``` evaluates to true as long as there's some text in ```phraseInput.value```.

Hopefully that makes sense and is not just more confusing!